### ISSUE TITLE
the master CLI binary is out of date

### ISSUE THREAD

**Issue by l1t1** [duckdb/duckdb#2754]
#### What happens?
the file is built at 20211204

#### To Reproduce
wget https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_cli-windows-amd64.zip

#### Environment (please complete the following information):
 - DuckDB Version: [e.g. 22]
 - DuckDB Client: [e.g. Python]

#### Before Submitting

- [x] **Have you tried this on the latest `master` branch?**
* **Python**: `pip install duckdb --upgrade --pre`
* **R**: `install.packages("https://github.com/duckdb/duckdb/releases/download/master-builds/duckdb_r_src.tar.gz", repos = NULL)`
* **Other Platforms**: You can find binaries [here](https://github.com/duckdb/duckdb/releases/tag/master-builds) or compile from source.

- [x] **Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?**

**Comment by l1t1** [comment]
other source code is out of date too. 
such as https://github.com/duckdb/duckdb/releases/download/master-builds/libduckdb-src.zip

**Comment by hannesmuehleisen** [comment]
Yes please built from GitHub for now, we disabled the master builds

**Comment by l1t1** [comment]
thanks. 
I use msvc 17, each run of nmake took about 10 min to complete.  any method to speed up it?

**Comment by kpm** [comment]
Please remove the "github master (bleeding edge)" option from this page, I was expecting to be able download it:  https://duckdb.org/docs/installation/index

**Comment by Mytherin** [comment]
I have updated the installation page to tell people to check out the release artifacts instead of the outdated master-builds
